### PR TITLE
Move HTML Tidy 'bare' option to the config

### LINKED ISSUE #359: "markup_repair_tidy" strips em-dash
When `markup_repair_tidy` is set to `true` and I try to post something with em-dash (`—`), it becomes en-dash or maybe even a hyphen. I suppose it has something to do with the locale but I did not test that.

### PR DESCRIPTION
**Pull request by rarjpg**
Fixes #359.

### CODE REVIEW

File: inc/config.php [hunk 1]
```diff
@@ -676,11 +676,15 @@
...
 
-	// Repair markup with HTML Tidy. This may be slower, but it solves nesting mistakes. Tinyboad, at the
+	// Repair markup with HTML Tidy. This may be slower, but it solves nesting mistakes. Tinyboard, at the
 	// time of writing this, can not prevent out-of-order markup tags (eg. "**''test**'') without help from
 	// HTML Tidy.
 	$config['markup_repair_tidy'] = false;
 
+	// Use 'bare' config option of tidy::repairString.
+	// It strips out smart quotes and em dashes, etc.
+	$config['markup_repair_tidy_bare'] = false;
```
Comment (ctrlcctrlv) [inline inc/config.php]: Making it configurable is good, changing the default is not. This should be `true`

Comment (rarjpg) [reply inc/config.php]: It should not because it strips correct chars from the text. Name me a "workflow" where that behavior is appropriate.

Comment (ctrlcctrlv) [reply inc/config.php]: Wait, you are saying that this option strips out &mdash;?

Comment (ctrlcctrlv) [reply inc/config.php]: So e.g.

> &ldquo;Hello&mdash;it&rsquo;s me you&rsquo;re looking for,&rdquo;

Becomes what? Certainly not:

>Helloits me youre looking for,

right?

Comment (rarjpg) [reply inc/config.php]: Oh, did not make that clear, sorry. It makes "-" from "—".

Comment (rarjpg) [reply inc/config.php]: And this looks like complete shit.

Comment (ctrlcctrlv) [reply inc/config.php]: Vichan is legacy software. Defaults should not be changed for aesthetic reasons. As it doesn't *strip* out em dashes, but rather ASCII-izes them (and other special chars), I cannot accept this patch unless you make the default the same as it is now. Just force push a version with the default set to true. Perhaps recommend changing it in a comment.

Comment (rarjpg) [reply inc/config.php]: Well, the comment is copied straight from the Tidy CLI docs, but okay.

**Comment by rarjpg** [comment]
@ctrlcctrlv

**Comment by rarjpg** [comment]
Oh please don't leave for two weeks again.

**Pull request merged**